src/countlyInterceptor.js

```diff
@@ -95,7 +95,7 @@ const ERROR_HANDLED_BY_OCLIF_KEY = "errorHandledByOclif";
 
 /**
  * @param {Error | unknown} errorOrUnknown
- * @returns
+ * @returns {never | Promise<void>}
```

Comment:
how could it be `never`? sounds strange to me. `unknown` made more sense

Reply:
never in typescript means that a function might not return anything. Here it's possible that this function just ends program execution and that's it